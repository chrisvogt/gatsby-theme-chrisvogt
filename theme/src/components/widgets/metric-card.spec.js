import React from 'react'
import renderer from 'react-test-renderer'
import MetricCard from './metric-card'

describe('MetricCard', () => {
  it('matches the snapshot', () => {
    const tree = renderer
      .create(
        <MetricCard title='Fake Metric' value='Fake Value' ready='true' />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
