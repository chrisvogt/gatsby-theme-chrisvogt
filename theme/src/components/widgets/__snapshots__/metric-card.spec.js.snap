// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MetricCard matches the snapshot 1`] = `
<div
  className="css-1yz7e9k"
>
  <span>
    Fake Value
  </span>
  Fake Metric
</div>
`;
